### PR TITLE
fix infinite loop 

### PR DESCRIPTION
**Pull request by LiboYu2**
1 fix 0 poolingfrequency issue
2 fix status parsing issue
3 fix timeout status update issue.

### CODE REVIEW

File: pkg/controllers/vrep/replication_status_reconciler.go [hunk 1]
```diff
@@ -233,6 +234,9 @@ func (r *ReplicationStatusReconciler) runReplicationStatus(ctx context.Context,
 	opts []replicationstatus.Option) (err error) {
 	timeout := vmeta.GetReplicationTimeout(r.Vrep.Annotations)
 	pollingFrequency := vmeta.GetReplicationPollingFrequency(r.Vrep.Annotations)
+	if pollingFrequency <= 0 {
```
Comment (cchen-vertica) [inline pkg/controllers/vrep/replication_status_reconciler.go:237]: Remove these 3 lines.

**Comment by roypaulin** [comment]
Change VReplicator concurrency to 3

**Comment by cchen-vertica** [comment]
The concurrent value also needs to be changed in "helm-charts/verticadb-operator/values.yaml", "helm-charts/verticadb-operator/README.md" and "config/manager/operator-envs",

**Comment by LiboYu2** [comment]
"helm-charts/verticadb-operator/values.yaml", "helm-charts/verticadb-operator/README.md" have been updated.
"config/manager/operator-envs" only has environmental variable keys and does not need to be updated.

**Review by cchen-vertica — APPROVED** [review]

**Pull request merged**